android/libraries/rib-base/src/main/java/com/badoo/ribs/core/view/ViewFactory.kt

```diff
@@ -2,4 +2,8 @@ package com.badoo.ribs.core.view
 
 import android.view.ViewGroup
 
+// TODO Replace ViewFactory with this
+interface ViewFactory2<D, T>: (D) -> (ViewGroup) -> T
```

Comment:
When do you plan to rid of suffix 2 in the name?
You don't want to spread `ViewFactory2` everywhere, right?

Reply:
This is only POC draft, will not look like this in the end. Please comment only on approach in general for now. Thanks.